library/core/src/num/mod.rs

```diff
@@ -970,12 +970,14 @@ pub enum FpCategory {
 
 #[doc(hidden)]
 trait FromStrRadixHelper: PartialOrd + Copy {
-    fn min_value() -> Self;
-    fn max_value() -> Self;
+    const MIN: Self;
     fn from_u32(u: u32) -> Self;
     fn checked_mul(&self, other: u32) -> Option<Self>;
     fn checked_sub(&self, other: u32) -> Option<Self>;
     fn checked_add(&self, other: u32) -> Option<Self>;
+    unsafe fn unchecked_mul(&self, other: u32) -> Self;
```

Comment:
minor: I assume `&self` here is copied from the `checked_` ones, but since the trait always requires `Copy` anyway, maybe just change these to `self`?  The indirection seems unhelpful.

Reply:
Nice, good spot!